src/components/SearchCard.tsx

```diff
@@ -33,7 +31,7 @@ export const SearchCard = memo(function SearchCard() {
   const { searchResults, setSearchResults } = useSearch()
   const searchResultsArray = useMemo(() => Object.values(searchResults), [searchResults])
   const id = useId()
-  const { analyzerData } = useData()
+  const { repodata2 } = useData()
```

Comment:
Is it time to rename this and remove the 2?

Reply:
ahh, yeah it is